cargo-miri/bin.rs

```diff
@@ -661,17 +661,6 @@ fn phase_cargo_miri(mut args: env::Args) {
         );
     }
     cmd.env("RUSTC_WRAPPER", &cargo_miri_path);
-    // Having both `RUSTC_WRAPPER` and `RUSTC` set does some odd things, so let's avoid that.
-    // See <https://github.com/rust-lang/miri/issues/2238>.
-    if env::var_os("RUSTC").is_some() && env::var_os("MIRI").is_none() {
-        println!(
-            "WARNING: Ignoring `RUSTC` environment variable; set `MIRI` if you want to control the binary used as the driver."
```

Comment:
We will not actually invoke this RUSTC, so I don't think it is good to just silently ignore that. We also warn about RUSTC_WRAPPER, see the code just above this one.

Reply:
`cargo` will invoke it...

Reply:
It shouldn't though, that would be a bug. That's exactly the issue that we are having currently.